javascript/javascript1/week4/js-week4.js

```diff
@@ -0,0 +1,111 @@
+let userName="";
+let toDoList=[];
+function getReply(command){
+    command=command.toLowerCase();
+    let resualt='command is not defined!';
```

Comment:
This variable's name is a little bit confuse to other programmers that might read your code. Maybe you meant **response** instead?